### ISSUE TITLE
Python 2 compatibility

### ISSUE THREAD

**Issue by sbraz** [thp/minidb#1]
Hi,
I see that tests fail with Python 2 and you mentioned that this module is made for Python 3. Does it mean it does not support Python 2? In this case I guess urlwatch doesn't either.
```
  File "minidb.py", line 70
    ...
    ^
SyntaxError: invalid syntax
```

**Comment by thp** [comment]
This is correct, minidb and urlwatch now require Python 3 and do not support Python 2 anymore. The first version of Python 3 has been released in 2008, and Python 3.3 (the version that urlwatch depends on), has been released in 2012.

This requirement is also mentioned in urlwatch:
https://github.com/thp/urlwatch/blob/master/README#L21

**Comment by sbraz** [comment]
OK, thanks for clearing this up!
